localstack-core/localstack/services/cloudformation/engine/v2/change_set_model_executor.py

```diff
@@ -355,12 +355,13 @@ def _execute_resource_action(
         )
         resource_provider = resource_provider_executor.try_load_resource_provider(resource_type)
         track_resource_operation(action, resource_type, missing=resource_provider is not None)
-        log_not_available_message(
-            resource_type,
-            f'No resource provider found for "{resource_type}"',
-        )
-        if resource_provider is None and not config.CFN_IGNORE_UNSUPPORTED_RESOURCE_TYPES:
-            raise NoResourceProvider
+        if resource_provider is None:
```

Comment:
This is a fix for a previous PR where I incorrectly logged the not available message regardless of whether we could load the resource provider or not.